### DISCUSSION TITLE
Welcome to automatic-ripping-machine Discussions!

### DISCUSSION THREAD

**Discussion by 1337-server** [automatic-ripping-machine/automatic-ripping-machine#493] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Snarf82** [comment]
Hello. I hope this is OK here. I am a almost total noob for anything non Windows OS, and i'm looking at using ARM. I've setup a Ubuntu 20.04 VM in Windows 10 which is all working ok. This is on a dedicated PC attached to my home network, fitted out with the optical drives and storage for the files.

Before i go any further, i'm really struggling to find out something:

Once ARM has ripped my media - how do i access it from my main Windows 10 PC on the network or the Windows 10 host PC?

It's the VM aspect that is confusing me.

Would it be better to dual boot this PC with Windows 10 and Ubuntu instead? I know each can read NTFS file system so i guess in theory my dedicated media drive will let me rip to it in Ubuntu and then read from it in Windows?

Thanks for any advice.

**Comment by 1337-server** [comment]
Depending on the software, you should be able to set up a shared folder from the Windows 10 to the VM. You can then change the arm.yaml config to set the output directories to the shared folder.

**Reply by Snarf82** [reply]
Thank you for the reply.

I couldn't get the shared folder in Hyper-V to work, so going to try VMware or VirtualBox instead - VMware looks to be simpler to setup!

**Comment by beaudans** [comment]
Hey there!

I'm trying to do a docker image pull install and running into some serious trouble.  I've gone through the directions and troubleshooting multiple times and can't figure out what I am doing wrong.  I have the install up and can access the web UI, but no ripping occurs when media is inserted.

Following the troubleshooting wiki, it doesn't look like I have udev on the host machine set up correctly, but I followed the guide exactly to place the files in the correct directories.  I can't find any references to ARM in the syslogs, but have both docker-arm.rules and docker_arm_wrapper.sh placed in the correct places.

Any help is appreciated!

Thank you

**Reply by 1337-server** [reply]
The current main docker image is bugged and no longer working fully. Use the dev image. Instruction are here.

https://github.com/automatic-ripping-machine/automatic-ripping-machine/issues/491#issuecomment-1114395193

The reason it's not been updated yet is that we are waiting for @shitwolfymakes to finalize the docker image. Once this is completed, the wiki and dockerhub README will be updated.

**Reply by beaudans** [reply]
still getting errors with dev image. log shows below:
[2022-05-30 15:00:33,816] DEBUG in routes: joblist
[2022-05-30 15:00:33,819] DEBUG in routes: {'success': False, 'mode': 'joblist', 'results': {}, 'arm_name': ''}
10.0.1.11 - - [30/May/2022 15:00:28] "GET /json?mode=joblist HTTP/1.1" 200 -

**Reply by 1337-server** [reply]
> still getting errors with dev image. log shows below: [2022-05-30 15:00:33,816] DEBUG in routes: joblist [2022-05-30 15:00:33,819] DEBUG in routes: {'success': False, 'mode': 'joblist', 'results': {}, 'arm_name': ''} 10.0.1.11 - - [30/May/2022 15:00:28] "GET /json?mode=joblist HTTP/1.1" 200 -

These are part of the normal logging functions, the `'success': False` does need updated to return true, though.

If you want fewer log updates, you can update the log level to INFO to reduce the amount.

**Comment by OperationNorthwoods** [comment]
does this work for the latest version of Ubuntu Server 22.04 LTS?

**Reply by 1337-server** [reply]
Base installation of ARM isn't currently working with 22.04 LTS

However, the docker image should work.
